libdrgn/dwarf_index.c

```diff
@@ -2686,3 +2818,21 @@ drgn_dwarf_index_iterator_next(struct drgn_dwarf_index_iterator *it,
 		*bias_ret = bias;
 	return NULL;
 }
+
+struct drgn_error *
+drgn_dwarf_index_iterator_next(struct drgn_dwarf_index_iterator *it,
+			       Dwarf_Die *die_ret, uint64_t *bias_ret)
+{
+	struct drgn_dwarf_index_die *die;
+	die = drgn_dwarf_index_iterator_next_die(it, true);
+	if (!die)
+		return &drgn_stop;
+	return drgn_dwarf_index_convert_die(die, die_ret, bias_ret);
+}
+
+inline struct drgn_dwarf_index_namespace *
```

Comment:
There's no point in making this inline since it can't be seen outside of `dwarf_index.c`. I think we might as well just move the definition of `struct drgn_dwarf_index_die` to `dwarf_index.h`.